### PR TITLE
[Feedback] [docrevamp] [devcommunity]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Doc revamp - please review

### CODE REVIEW

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -2,28 +2,38 @@
...
...   |<ol><li> Questions can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.  Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. New users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask) for more details. </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:18]:
```suggestion
|         **Community support**          |<ol><li> Questions can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.  Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. For guidance on how to ask your queries, users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask). </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
```

File: msteams-platform/feedback.md [hunk 2]
```diff
@@ -2,28 +2,38 @@
...
+|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams) using the `microsoft-teams` tag.                                                                                                                                                                                                                       |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:19]:
```suggestion
|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |
```

File: msteams-platform/feedback.md [hunk 3]
```diff
@@ -2,28 +2,38 @@
...
+|            **Bot & Messaging Extension SDK issues/suggestions**             |      Submit issues and feature requests to the SDK repository for your bot's language ([C#](https://github.com/Microsoft/botbuilder-dotnet/), [Javascript](https://github.com/Microsoft/botbuilder-js), or [Python](https://github.com/Microsoft/botbuilder-python)). How-to questions about the SDK can be posted at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams) using the `botframework & microsoft-teams` tag.                                                                                            |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:20]: Use the `botframework & microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/botframework%20microsoft-teams).

File: msteams-platform/feedback.md [hunk 4]
```diff
@@ -2,28 +2,38 @@
...
+| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |
+| **Features suggestions**             |      Make your feature's suggestions on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:22]: Do features make suggestions?
```suggestion
| **Features suggestions**             |      Make your suggestions for a feature on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
```

File: msteams-platform/feedback.md [hunk 5]
```diff
@@ -2,28 +2,38 @@
...
+|       **Documentation updates**        | Select the **Edit** link in an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                                      |
+|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev) if you are logging in for the first time. Select **JOIN ROOM**, if you are already logged in. |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:25]: check the punctuations.
replace "logging" with "signing"

File: msteams-platform/feedback.md [hunk 6]
```diff
@@ -2,28 +2,38 @@
...
+|          **General questions**         |          General questions about the Microsoft Teams developer platform must be sent to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned and use email only if none of the above are applicable.                                                                                                                                                                          |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:26]:
```suggestion
|          **General questions**         |          Send general questions about the Microsoft Teams developer platform to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned, and use email only if no other mode of communication is applicable.                                                                                                                                                                          |
```

File: msteams-platform/feedback.md [hunk 7]
```diff
@@ -2,28 +2,38 @@
...
 
 ## Product Support channels
-|            **Support type**            |               **Contact**                                                                                  |
+Support channels to submit your Teams product related questions:
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:29]: The following table lists the support channels to submit Teams product related queries:

File: msteams-platform/feedback.md [hunk 8]
```diff
@@ -2,28 +2,38 @@
...
+
+|            **Support type**            |               **Submit your issues or queries**                                                                                  |
```
Comment (KirtiPereira) [inline msteams-platform/feedback.md:31]: Why submit your issues or queries, as a table header? Are we not giving them where or whom to contact?
Suggest using Contact.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - please validate and incorporate the comments.

File: msteams-platform/feedback.md [hunk 9]
```diff
@@ -1,29 +1,36 @@
...
...teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams). Stack Overflow has guidelines, such as requiring a descriptive title, a complete and concise problem statement, and sufficient details to reproduce your issue. Feature requests or overly broad questions are off-topic. For guidance on how to ask your queries, users must visit the [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask). </li>                                                                                                                                                                       <li>  Questions can also be posted at [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) using the `office-teams-app-dev` tag. We love Stack Overflow and our extensive support is assured for customers who ask questions there. However, Stack Overflow has specific criteria about what questions are appropriate for the community and [Microsoft Q&A](/answers/topics/office-teams-app-dev.html) has an open policy regarding this.  </li> </ol>                                                                                                  |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:15]: Please rephrase as follows:

You can post questions on one of the following:

- [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams): Use the microsoft-teams tag to post the questions.
   > [!NOTE]
   > Ensure that you follow the Stack Overflow guidelines such as provide a descriptive title, a complete and concise problem statement, and sufficient detail to reproduce your issue. Feature requests or overly broad questions are off-topic. if you are a new user, for more information, see [Stack Overflow Help Center](https://stackoverflow.com/help/how-to-ask).
   
- Rephrase in the similar way for point 2.

File: msteams-platform/feedback.md [hunk 10]
```diff
@@ -1,29 +1,36 @@
...
+|        **Tab SDK issues/suggestions**        |  Submit issues and feature requests to the SDK repository for [Microsoft Teams JavaScript Library](https://github.com/OfficeDev/microsoft-teams-library-js/issues). Use the `microsoft-teams` tag to post how-to questions about the SDK at [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoft-teams).                                                                                                                                                                                                                       |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:16]: Global: Please make the list in alphabetical order

Comment (surbhigupta12) [inline msteams-platform/feedback.md:16]: Change to:

Tab SDK issues or suggestions

Comment (surbhigupta12) [inline msteams-platform/feedback.md:16]: Change the tone to active

File: msteams-platform/feedback.md [hunk 11]
```diff
@@ -1,29 +1,36 @@
...
+| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |
+| **Features suggestions**             |      Make your suggestions for a feature on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:19]: You can post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.

File: msteams-platform/feedback.md [hunk 12]
```diff
@@ -1,29 +1,36 @@
...
-|         **Microsoft Teams product issues/failures**          | Tenant Admin can reach out to [Microsoft 365 Product Support](/microsoft-365/admin/contact-support-for-business-products).                                                            |
-|        **General questions about Microsoft Teams**        |  Please post your question on [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).               |                                                                                                                                                         
+|         **Microsoft Teams product issues and failures**          | Tenant Admin can reach out to [Microsoft 365 Product Support](/microsoft-365/admin/contact-support-for-business-products).                                                            |
+|        **General questions about Microsoft Teams**        |  Submit your questions on [Microsoft Teams Community](https://answers.microsoft.com/en-us/msteams/forum).               |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:31]: Please maintain parallelism across the table.

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Provided comments, please look into them.

File: msteams-platform/feedback.md [hunk 13]
```diff
@@ -1,29 +1,35 @@
...
+|  **Documentation issues**  |        Choose `This page` option in the **Submit and view feedback** for section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                                            |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:17]:
```suggestion
|  **Documentation issues**  |        Select `This page` in the **Submit and view feedback** section available at bottom of the documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                                            |
```

File: msteams-platform/feedback.md [hunk 14]
```diff
@@ -1,29 +1,35 @@
...
+|  **Documentation updates**           |     Select the **Edit** link in an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                           |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:18]:
```suggestion
|  **Documentation updates**           |     Select **Edit** in the article you want to update and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                           |
```

File: msteams-platform/feedback.md [hunk 15]
```diff
@@ -1,29 +1,35 @@
...
+|       **Features suggestions**       |                                                                                                                                                                      You can post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.                                                                                                                                                                      |
```
Comment (surbhigupta12) [inline msteams-platform/feedback.md:19]:
```suggestion
|       **Features suggestions**       |                                                                                                                                                                      Post features suggestions on (https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform.                                                                                                                                                                      |
```
```

**Review by surbhigupta12 — CHANGES_REQUESTED** [review]
Reviewed the file.

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR.

**Pull request merged**